SetupDataPkg/Tools/VariableList.py

```diff
@@ -1178,24 +1178,29 @@ def read_csv(schema, csv_path):
             knob.value = knob.format.string_to_object(knob_value_string)
 
 
-def write_csv(schema, csv_path, subknobs=True):
+def write_csv(schema, csv_path, full, subknobs=True):
     with open(csv_path, 'w', newline='') as csv_file:
         writer = csv.writer(csv_file)
 
+        # delta_vlist is a tuple of name_list, var_list
+        delta_vlist = get_delta_vlist(schema)

```

Comment:
Very minor, but if you don't think you'll need the `var_list` from `get_delta_vlist()` here, it would be a little easier to read to track `name_list` as a standalone variable instead of as `delta_vlist[0]`.

Something like:

```python
        name_list = get_delta_vlist(schema)[0]
```

Reply:
Agreed, updated.